cargo-shuttle/src/lib.rs

```diff
@@ -1639,8 +1768,7 @@ impl Shuttle {
 }
 
 fn check_version(runtime_path: &Path) -> Result<()> {
-    let valid_version = semver::Version::from_str(VERSION)
-        .context("failed to convert cargo-shuttle version to semver")?;
+    let my_version = semver::Version::from_str(VERSION).unwrap();
```

Comment:
Why drop the `.context(` call? I think it is better than failing with a panic.

Reply:
the constant is always a valid semver.